README.rst

```diff
@@ -35,14 +37,26 @@ using::
 Installation
 ------------
 
-1. Install ``sectiondoc`` from pypi using pip::
+Install ``sectiondoc`` from pypi using pip::
 
     $ pip install sectiondoc
 
-2. Add sectiondoc to the extensions variable of your sphinx ``conf.py``::
+Usage
+-----
+
+Add sectiondoc to the extensions variable of your sphinx ``conf.py``::
 
     extensions = [
         ...,
         'sectiondoc',
```

Comment:
If I read the warning in `sectiondoc.__init__` correctly, this usage is deprecated. If so, it should not be the first instruction (and probably shouldn't be in the README at all. It should be noted in the main documentation as a deprecated interface.